motor/vault/credentials_resolver/credentials_resolver.go

```diff
@@ -0,0 +1,68 @@
+package credentials_resolver
+
+import (
+	"context"
+	"errors"
+
+	"github.com/rs/zerolog/log"
+	"go.mondoo.com/cnquery/motor/vault"
+)
+
+type Resolver interface {
```

Comment:
Interface should move into the vault package into vault.go

Reply:
done